pkg/feature/templates/servicemesh/kserve/grpc-envoyfilter-temp-fix.tmpl

```diff
@@ -0,0 +1,36 @@
+# Temporary workaround for https://issues.redhat.com/browse/RHOAIENG-165
+apiVersion: networking.istio.io/v1alpha3
+kind: EnvoyFilter
+metadata:
+  name: grpc-only
+  namespace: {{ .ControlPlane.Namespace }}
+  labels:
+    app.opendatahub.io/kserve: "true"
+    app.kubernetes.io/part-of: kserve
```

Comment:
```suggestion
    app.kubernetes.io/part-of: kserve
    opendatahub.io/related-to: RHOAIENG-165
```

This can help in automated cleanup moving forward. We have similar temporary filter already, which follows similar labeling https://github.com/opendatahub-io/opendatahub-operator/blob/incubation/pkg/feature/templates/servicemesh/kserve/envoy-oauth-temp-fix.tmpl

Reply:
Done